### ISSUE TITLE
Annotate Swaps with parties in tactical vote recommendations

### ISSUE THREAD

**Issue by baob** [swapmyvote/swapmyvote#773]
Examples:

Users preferred party is green, recommendation is green, we say ":tick: Good swap (according to \<recommendation site\>)
Users preferred party is green, recommendation is Any, we say ":tick: Good swap (according to \<recommendation site\>)
Users preferred party is green, recommendation is LibDem, we say nothing. 

#772 MUST be done first, presently tactical vote recommendations are purely text, we need to match that to parties specified by users to make the annotation

**Comment by baob** [comment]
Overlaps with #667

**Comment by baob** [comment]
Suggestions from chat elsewhere:
 - [ ] use full party names not abbreviations
 - [ ] show ''TV site had no advice' as appropriate
 - [ ] just have the tick and no "good or bad".
 - [ ] a fake RW campaign (or two) FOR USE ONLY IN DEV to balance out the LW ones.

**Comment by baob** [comment]
The NI problem

![image](https://github.com/swapmyvote/swapmyvote/assets/55932/ebad6a23-6b68-4230-b868-ba3b7c36a812)

**Comment by aspiers** [comment]
@baob  I suggest filing a separate issue for that NI issue - while it's not great it doesn't seem like a launch blocker.
